### ISSUE TITLE
Tab Preferences and Converting tabs to spaces

### ISSUE THREAD

**Issue by samirotiv** [lee-dohm/indentation-indicator#11]
Hi,
If it's not too difficult, could you please extend the program to set the current tab width, and also to convert tabs to spaces & the other way round like Sublime Text?

I believe there's already an Atom plugin called tab-control that sets the tab width, and another called tabs-to-spaces that converts indentation. If you could hook up the status bar tool to these 2 it would be great.!!

Warm regards,
Samir.

**Comment by lee-dohm** [comment]
Yep, I wrote the [tabs-to-spaces package](https://atom.io/packages/tabs-to-spaces). I'll see what I can do now that there is a method of packages providing services to other packages.

**Comment by JesseLeite** [comment]
Great package, thank you!

At the very least, when running Tabify or Untabify from tabs-to-spaces, update indentation-indicator in status bar :)

It would be cool if this were all available in one package though.  Click indentation-indicator in status bar to change, much like how Sublime handles.
